package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
+    "start:local": "nodemon index.js --baseUrl http://0.0.0.0:4000",
```

Comment:
@zelima here we are just overriding `baseUrl` property of settings. By default it is set to `baseUrl: 'https://staging.datapackaged.com'`.